client/src/components/TenseHeaderMobile.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Select from 'react-select';
+
+
+const INDICATIVE_TENSES = [
+  "Presente",
+  "Presente - Imperativo",
+  "Presente perfecto",
+  "Pretérito",
+  "Futuro",
+  "Futuro perfecto",
+  "Pluscuamperfecto",
+  "Imperfecto",
+  "Pretérito anterior",
+  "Condicional",
+  "Condicional perfecto"
+];
+
+
+const customStyles = {
+  option: (provided, state) => ({
+    ...provided,
+    background: state.isSelected ? '#4cd0ba' : 'white',
+    color: state.isSelected ? 'white' : 'black'
+  }),
+};
+
+export default ({activeTenses, toggleTense, tenseOptions, moodSuffix=''}) => (
+  <div>
+    <Select
+      value={moodSuffix.length ? 'Subjunctive' : 'Indicative / Imperative'}
+      onChange={toggleTense}
+      isMulti={true}
+      hideSelectedOptions={false}
+      styles={customStyles}
+      value={ 
+        Object.keys(activeTenses)
+        .filter(at => activeTenses[at] && tenseOptions.indexOf(at) != -1)
```

Comment:
might make sense to incorporate React propTypes -- its a bit confusing that activeTenses is an object and tenseOptions is an array

Reply:
will write a TODO but don't feel like it rn